### PR TITLE
Fix CI Failures

### PR DESCRIPTION
**Pull request by alzimmermsft**
Fixes mismatching version tags and missing README section.

### CODE REVIEW

**Comment by alzimmermsft** [comment]
@azabbasi mind taking a look

File: sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/ModelsRepositoryAPIImplBuilder.java [hunk 1]
```diff
@@ -190,7 +190,7 @@ public ModelsRepositoryAPIImplBuilder addPolicy(HttpPipelinePolicy customPolicy)
      * @return an instance of ModelsRepositoryAPIImpl.
      */
     public ModelsRepositoryAPIImpl buildClient() {
-        if(apiVersion == null){
+        if (apiVersion == null) {
```
Comment (alzimmermsft) [inline sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/ModelsRepositoryAPIImplBuilder.java:193]: @srnagar is this an issue with codegen?

Comment (srnagar) [reply sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/ModelsRepositoryAPIImplBuilder.java:193]: @alzimmermsft are you referring to the formatting?

Comment (alzimmermsft) [reply sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/ModelsRepositoryAPIImplBuilder.java:193]: Yeah, it was being flagged by Checkstyle linting

Comment (srnagar) [reply sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/ModelsRepositoryAPIImplBuilder.java:193]: yeah, I'll get this fixed in autorest.

File: sdk/modelsrepository/azure-iot-modelsrepository/src/test/java/com/azure/iot/modelsrepository/DtmiConventionTests.java [hunk 2]
```diff
@@ -53,7 +49,24 @@ public void getModelUriTests(String repository, String expectedUri) throws URISy
...
+            Arguments.of("https://localhost/REPOSITORY",
+                "https://localhost/REPOSITORY/dtmi/com/example/thermostat-1.json"),
+            Arguments.of("file:///path/to/repository/",
+                "file:///path/to/repository/dtmi/com/example/thermostat-1.json"),
+            Arguments.of("file://path/to/RepoSitory", "file://path/to/RepoSitory/dtmi/com/example/thermostat-1.json")
+
+            // TODO: These were disabled as they fail in Linux and macOS, likely due to '\' in the URI.
+//            Arguments.of("C:\\path\\to\\repository\\",
+//                "file:///C:/path/to/repository/dtmi/com/example/thermostat-1.json"),
+//            Arguments.of("\\\\server\\repository", "file:////server/repository/dtmi/com/example/thermostat-1.json")
```
Comment (alzimmermsft) [inline sdk/modelsrepository/azure-iot-modelsrepository/src/test/java/com/azure/iot/modelsrepository/DtmiConventionTests.java:66]: @azabbasi these we disabled as they are failing in Linux and macOS, likely due to \ being in the URI

Comment (azabbasi) [reply sdk/modelsrepository/azure-iot-modelsrepository/src/test/java/com/azure/iot/modelsrepository/DtmiConventionTests.java:66]: I am working on fixing these in the PR I have open
you can merge this in if they are causing issues and cannot wait for the fix.

**Review by azabbasi — APPROVED** [review]

**Pull request merged**